ports/sqlgen/vcpkg.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "sqlgen",
+  "version": "0.1.0",
+  "description": "sqlgen is an ORM and SQL query generator for C++-20, similar to Python's SQLAlchemy/SQLModel or Rust's Diesel.",
+  "homepage": "https://github.com/getml/sqlgen/",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "reflectcpp",
+      "version>=": "0.18.0"
```

```suggestion
      "name": "reflectcpp"
```

Don't add version constraints to packages, let users control versions in their projects directly.

---

But it won't work with older versions of reflectcpp.